docs/C2D.md

```diff
@@ -56,7 +56,10 @@ These internal properties can be used to interact with any aspect of the Web's [
 | `height` | number | the [height](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/height) of the canvas, an alias for `C2D.canvas.height` |
 | `fill` | string | the current [fill style](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/fillStyle) of the drawing context, an alias for `C2D.ctx.fillStyle` |
 | `stroke` | string | the current [stroke style](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/strokeStyle) of the drawing context, an alias for `C2D.ctx.strokeStyle` |
-
+| `fontSize` | number | the font size in pixels of any font drawn on the canvas. Allows easier writing to [`C2D.ctx.font`](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/font) Defaults to 48. |
+| `fontStyle` | string | the actual style of text drawn on the canvas. Allows for easier writing to [`C2D.ctx.font`](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/font). Defaults to `'serif'`|
+|  `fontAlign` | string | the [text align](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/textAlign) for text drawn on the canvas, an alias for `C2D.ctx.textAlign`. Must be either 'left', 'right, 'center', 'start', or 'end', or an error will be thrown. See the linked Mozilla docs for more on each option. Is set to `'center'`.
+| `fontBaseline` | string | the [text baseline](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/textBaseline) of text drawn on the canvas, an alias for `C2D.ctx.textBaseline`. Must be either 'top', 'hanging', 'middle', 'alphabetic', 'ideographic', or 'bottom'. See the linked Mozilla docs for more on each option. Is set to `'middle'` by default.
 #### `ellipse(x, y, w, [h])`
```

Comment:
in markdown (`.md`) files u need to include a line break between this line && the table before it, otherwise the line gets interpreted as part of the table (rather than as a heading as it should be)